.yamllint

```diff
@@ -0,0 +1,2 @@
+ignore: |
+    .github
```

Comment:
Consider changing your text editor to insert final newlines.
This causes issues with some tooling.